### DISCUSSION TITLE
SlicerANTs extension

### DISCUSSION THREAD

**Discussion by simonoxen** [ANTsX/ANTs#1374] [General]
Hi, in case there's interest, I just wanted to point to the SlicerANTs extension I've been working on.

It implements antsRegistration in 3D Slicer with an UI to set parameters.

Now available from Slicer 5 Extensions Manager.

See here source: [https://github.com/netstim/SlicerANTs](https://github.com/netstim/SlicerANTs)

Thanks!

**Comment by ntustison** [comment]
Awesome, @simonoxen .  Thanks for this and letting us know.
